docs/templates/story.rst.j2

```diff
@@ -181,35 +182,35 @@
 
 {# Links pointing to step core implementation #}
 {% elif link.target | match('^/tmt/steps(?:/__init__.py)?$') %}
-{{ emit_tmt_repo_link(link, '/tmt/(steps)(?:/__init__.py)?') }}
+{{ emit_tmt_repo_link(link) }}
 
 {# Links pointing to step implementations #}
 {% elif link.target | match('^/tmt/steps/([a-z_]+)(?:$|(?:/__init__.py))$') %}
-{{ emit_tmt_repo_link(link, '/tmt/steps/([a-z_]+)(?:$|(?:/__init__.py))') }}
+{{ emit_tmt_repo_link(link) }}
 
 {# Links pointing to step plugins #}
 {% elif link.target | match('^/tmt/steps/([a-z_]+/[a-z_]+)\\.py$') %}
 {{ emit_tmt_plugin_link(link) }}
 
 {# Links pointing to other tmt code #}
 {% elif link.target.startswith('/tmt') %}
-{{ emit_tmt_repo_link(link, '/(.*)') }}
+{{ emit_tmt_repo_link(link) }}
 
 {# Links pointing to stories #}
 {% elif link.target | match('^/stories') %}
 * {{ printable_relation(link) }} :ref:`{{ link.target }}`
 
 {# Links pointing to tests #}
 {% elif link.target | match ('^/tests') %}
-{{ emit_tmt_object_link(link, '.fmf') }}
+{{ emit_tmt_object_links(link) }}
 
 {# Links pointing to plans #}
 {% elif link.target | match ('^/plans') %}
-{{ emit_tmt_object_link(link, '.fmf') }}
+{{ emit_tmt_object_links(link) }}
 
 {# Links pointing to websites #}
 {% elif link.target | match('^https?://') %}
-* {{ printable_relation(link) }} `{{ link.target}} <{{ link.target }}>`_
+* {{ printable_relation(link) }} `{{ link.target}} <{{ link.target }}>`__
```

Comment:
```suggestion
* {{ printable_relation(link) }} `{{ link.target}} <{{ link.target }}>`_
```

This underscore also does not seems to be needed.

Reply:
Fixed in 841a8924